packages/fast-usdc-contract/src/exos/advancer.ts

```diff
@@ -255,16 +256,20 @@ export const prepareAdvancerKit = (
           result: undefined,
           ctx: AdvancerVowCtx & { tmpSeat: ZCFSeat },
         ) {
-          const { poolAccount, intermediateRecipient, settlementAddress } =
-            this.state;
+          const { poolAccount, settlementAddress } = this.state;
           const { destination, advanceAmount, tmpSeat, ...detail } = ctx;
           tmpSeat.exit();
           const amount = harden({
             denom: usdc.denom,
             value: advanceAmount.value,
           });
+          const intermediateRecipient = getNobleICA().getAddress();
```

Comment:
What's the type here - i think it's `Vow<CosmosChainAddress>` instead of `CosmosChainAddress`?

Reply:
No, it's local so it's `CosmosChainAddress`.